src/test/java/com/bloxbean/cardano/client/backend/api/helper/FeeCalculationServiceTest.java

```diff
@@ -114,7 +110,7 @@ void testCalculateFeeSimplePaymentTransaction() throws ApiException, IOException
     void testCalculateFeeMultiplePaymentTransactions() throws ApiException, IOException, AddressExcepion, CborSerializationException {
 
         String receiver1 = "addr_test1qqwpl7h3g84mhr36wpetk904p7fchx2vst0z696lxk8ujsjyruqwmlsm344gfux3nsj6njyzj3ppvrqtt36cp9xyydzqzumz82";
-        String receiver2 = "addr_test1qqwpl7h3g84mhr36wpetk904p7fchx2vst0z696lxk8ujsjyruqwmlsm344gfux3nsj6njyzj3ppvrqtt36cp9xyydzqzumz82";
+        String receiver2 = "addr_test1qpchc7f6zln5c2ruarssn9p8vulhhn3rk7cvz4lm05eq2xwx8e7tf5l25n4ek3gch8tnj4k5z4236fvy6yndrusf7x0sv287t5";
```

Comment:
@satran004 in order to fix the test maintaining the rationale, I've changed the receiver address to something different, this demonstrated the backward compatibility of my changes.

Reply:
Thanks. Makes sense. Not sure why both were same before :)